vue-reviews/src/components/knowledgesharing/KnowldedgeSharing.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <md-layout md-vertical-align="start" v-if="!activeUserGetter.isAnonymous" id="kn-tab">
+    <kn-header :items="items"
+               :filters="filters"
+               @kn-item-submitted="submitData"
+               @kn-filter-removed="removeFilter"
+               @kn-filters-cleared="removeAllFilters"
+               @kn-search-filtered="addToFilter"
+               @kn-search-applied="(value) => {search = value}"
+               @kn-sorted-newest="(isNewestFirst) => {sortNewestFirst = isNewestFirst}"
+               @kn-filtered-date="(date) => {dateFilter = date} ">
+    </kn-header>
+    <kn-content :items="filteredItems"
+                :filters="filters"
+                @kn-filter-applied="addToFilter">
+    </kn-content>
+  </md-layout>
+  <md-layout v-else>
+    Please log in to see the data!
+  </md-layout>
+</template>
+<script>
+  import firebase from 'firebase';
+  import {mapActions, mapGetters} from 'vuex';
+  import {FBApp} from '@/data/firebase-config';
+  import MdLayout from "../../../node_modules/vue-material/src/components/mdLayout/mdLayout";
```

Comment:
This should be already accessible, no need to import it manually again, please let me know if you run into issues with this element.